Setup.hs

```diff
@@ -82,13 +82,14 @@ getLLVMConfig confFlags = do
   return $ getProgramOutput verbosity program
 
 addToLdLibraryPath :: String -> IO ()
-addToLdLibraryPath path = do
-  let (ldLibraryPathVar, ldLibraryPathSep) =
-        case buildOS of
-          OSX -> ("DYLD_LIBRARY_PATH",":")
-          _ -> ("LD_LIBRARY_PATH",":")
-  v <- try $ getEnv ldLibraryPathVar :: IO (Either SomeException String)
-  setEnv ldLibraryPathVar (path ++ either (const "") (ldLibraryPathSep ++) v)
+addToLdLibraryPath path =
```

Comment:
Does this change break on a non-Nix installation? That's a third scenario we also need to consider :sweat_smile:

Reply:
Good question. Let me give it a try, if it breaks non-Nix then I guess we can revert these file changes and apply them changes as a patch for Nix.

Reply:
That would work! Let me know how it goes.